backend/src/db.py

```diff
@@ -49,3 +49,8 @@ def read_item_records(db_name: str) -> list[ItemRecord]:
     records = session.query(ItemRecord).all()
     session.close()
     return records
+def read_analysis_records(db_name: str) -> list[AnalysisRecord]:
```

Comment:
```suggestion
    return records
    
def read_analysis_records(db_name: str) -> list[AnalysisRecord]:
```

フォーマッタ効いてたらこの間空くと思うんだが

Reply:
範囲選択のステージでコミット上げてたせいで、上手く空白がコミットに入ってなかったみたい